botcraft/include/botcraft/AI/Tasks/InventoryTasks.hpp

```diff
@@ -79,11 +79,11 @@ namespace Botcraft
     /// @param client The client performing the action
     /// @param item_name The item to place
     /// @param pos The location where we want to place the block
-    /// @param face The face on which the block is placed
+    /// @param face The face on which the block is placed. Of not specified the optimal placing face will be automatically detected
```

Comment:
Of -> If
automatically detected as the closest one to the player facing direction